src/Test.UnitTests.Sarif/TestData/PrereleaseCompatibilityTransformer/Inputs/OneRunWithBasicInvocation.sarif

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-2.1.0-beta.2",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "CodeScanner",
+          "semanticVersion": "2.1.0"
+        }
+      },
+      "invocations": [
+        {
+          "commandLine": "CodeScanner @collections.rsp",
+          "responseFiles": [
+            {
+              "uri": "collections.rsp",
+              "index": 0
+            }
+          ],
+          "startTimeUtc": "2016-07-16T14:18:25.000Z",
+          "endTimeUtc": "2016-07-16T14:19:01.000Z",
+          "toolExecutionSuccessful": false,
```

Comment:
>"toolExecutionSuccessful": false, [](start = 10, length = 33)

i needed these new cases to ensure my new code paths are executed

Reply:
(also note schema is beta.2)

---
In reply to: [279148057](https://github.com/Microsoft/sarif-sdk/pull/1465#discussion_r279148057) [](ancestors = 279148057)